components/gitpod-protocol/src/teams-projects-protocol.ts

```diff
@@ -163,6 +163,7 @@ export interface OrgMemberInfo {
     avatarUrl?: string;
     role: TeamMemberRole;
     memberSince: string;
+    ownedByOrganizationId?: string;
```

Comment:
Could we solve this by referencing the OrgID on the Member (after all, the member is effectively just a (TeamID, UserID, Role) tuple, so it does kind of contain this. 

Given we only use it change what we show in the UI, could we not keep the UI the same and when actioned, returned a friendly message saying the this is not possible? It would reduce the UI complexity and reduce cases

In general, it feels odd. Is there ever the case where the Org ID of the `ownedByOrganizationId` would be different?

Reply:
Even if we want to disallow org-owned users to join other orgs forever, we need to know if `user.organizationId` is set or not

Reply:
That would suggest it's a property of the User, not the TeamMembership, or the Organization, no?

Reply:
Yes, but it is expensive to fetch. We can make it a boolean `isOwned` which would make a bit more sense on the relationship object.